authentication/serializers.py

```diff
@@ -0,0 +1,64 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User
+from .mixins import JWTTokenMixin
+from rest_framework.validators import UniqueValidator
+from django.contrib.auth.hashers import make_password
+
+
+class UserSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = (
+            'id', 'first_name', 'last_name', 'username', 'email', 'password', 'is_staff', 'full_name', 'is_active'
+        )
+        read_only_fields = ('is_active', 'is_staff')
+        extra_kwargs = {
+            'password': {'write_only': True},
+        }
+
+    def get_full_name(self, obj):
+        if obj.first_name and obj.last_name:
+            return '{} {}'.format(obj.first_name, obj.last_name)
+        return ""
+
+
+class UserSerializerWithJWT(JWTTokenMixin, UserSerializer):
```

Comment:
conventionally main models come first.
 class UserSerializerWithJWT(UserSerializer, JWTTokenMixin):

Reply:
Mixin comes first:

ref: https://github.com/encode/django-rest-framework/blob/master/rest_framework/generics.py
`class RetrieveAPIView(mixins.RetrieveModelMixin,
                      GenericAPIView):
    """
    Concrete view for retrieving a model instance.
    """
    def get(self, request, *args, **kwargs):
        return self.retrieve(request, *args, **kwargs`